src/shipit_static_analysis/shipit_static_analysis/report/mozreview.py

```diff
@@ -25,6 +25,7 @@
 COMMENT_SUCCESS = '''
 C/C++ static analysis didn't find any defects in this patch. Hooray!
 '''
+BUG_REPORT_URL = 'http://bit.ly/2y9N9Vx'
```

Comment:
Nit: We could create a mzl.la link with a human readable name.

Can we use https instead of http?

Reply:
> Nit: We could create a mzl.la link with a human readable name.

Cool suggestion! Please suggest human readable names for our two links.

> Can we use https instead of http?

I agree.